src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
-body {
+@import './blocks/variables';
+@import './blocks/card';
+@import './blocks/stars';
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+.body {
   margin: 0;
+  padding: 0;
```

Comment:
The body doesn't have a padding
```suggestion
```

Reply:
Not fixed 
<img width="1085" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/f0f2175e-54c1-44c7-a335-24f8ae1d785a">